techniques/fileDistribution/fileTemplate/1.0/fileTemplate.st

```diff
@@ -0,0 +1,146 @@
+# Take a template from a local source and expand it
+#   -Copy the template in /var/rudder/tmp/
+#   -Expand the template
+#   -Take options in considerations(like perms)
+#   -Reporting
+
+bundle agent fileTemplate {
+  vars:
+    #Tracking key
+    &TRACKINGKEY:{key | "trackingkey[&i&]" string => "&key&";}&
+
+    #Place to copy the template on the local node
+    "temp" string => "/var/rudder/tmp";
```

Comment:
I think we might use or want to use this folder for other purposes, and I would prefer a subfolder for templates. Plus current permissions of this folder are 755, which is not what we expect for user policies.
@peckpeck What do you think about that?